### ISSUE TITLE
Line breaks in table cells

### ISSUE THREAD

**Issue by calmh** [DavBfr/dart_pdf#43]
At the moment, the table is a bit tricky to use with strings because it breaks if a cell overflows. That is, the following example:

```
List<int> testPDF() {
  final Document pdf = Document(deflate: zlib.encode);
  pdf.addPage(
    MultiPage(
      pageFormat:
          PdfPageFormat.a4.copyWith(marginBottom: 1.5 * PdfPageFormat.cm),
      build: (Context context) => <Widget>[
            Header(level: 0, text: 'Table Test'),
            Table.fromTextArray(context: context, data: <List<String>>[
              <String>['Field', 'A', 'B', 'C'],
              <String>[
                'Foo',
                'This is a quite long field which may require breaking',
                'This is a quite long field which may require breaking',
                'This is a quite long field which may require breaking',
              ],
            ]),
          ],
    ),
  );
  return pdf.document.save();
}
```

fails to render with

```
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The method 'toDouble' was called on null.
Receiver: null
Tried calling: toDouble()
#0      Object.noSuchMethod (dart:core/runtime/libobject_patch.dart:50:5)
#1      double.+ (dart:core/runtime/libdouble.dart:17:23)
#2      RichText.layout  ../…/widgets/text.dart:233
#3      Align.layout  ../…/widgets/basic.dart:241
#4      Padding.layout  ../…/widgets/basic.dart:75
#5      StatelessWidget.layout  ../…/widgets/widget.dart:99
#6      Table.layout  ../…/widgets/table.dart:228
#7      MultiPage.generate (file:///Users/jb/sdk/flutter/.pub-cache/hosted/pub<…>
```

**Comment by DavBfr** [comment]
Yes, it is not the best widget of the library, but at least it shouldn't crash.

**Comment by plake876** [comment]
Bug is fixed in version 1.3.6

**Comment by DavBfr** [comment]
Thanks.
